puppet/zulip/manifests/postgres_appdb_tuned.pp

```diff
@@ -23,6 +23,13 @@
     group  => 'postgres',
   }
 
+  if $::osfamily == 'redhat' {
+    exec { "sudo /usr/pgsql-${zulip::base::postgres_version}/bin/postgresql-${zulip::base::postgres_version}-setup initdb":
+      require => Package[$zulip::postgres_appdb_base::postgresql],
+      creates => "$pg_datadir/pg_stat"
+    }
+  }
```

Comment:
Can you add a comment explaining why this step is required on Red Hat (I assume it's to initialize an empty database like the Debian postgres installer does?)

Reply:
Comment added.